### PR TITLE
feat: polynomial chain & polynomial sepolia

### PR DESCRIPTION
**Pull request by Aniket965**

### CODE REVIEW

**Review by SgtJG0321 — COMMENTED** [review]
👍

**Comment by Dickson0101-ai** [comment]
> Hey @ligi, i am from polynomial. we are going to be live soon can you review this if you have time

Yes

File: _data/icons/polynomial.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+    {
+      "url": "ipfs://bafybeiadb4izxxptk3wy34cam3k2rxgqb2vrzxtltqnp3vwdvd2igwoghu",
+      "width": 500,
+      "height": 500,
+      "format": "png"
```
Comment (macvts) [inline _data/icons/polynomial.json:6]: error due to the expected format and the format from url is not match 

change this format to jpg

File: _data/icons/polynomialSepolia.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[
+    {
+      "url": "ipfs://bafybeidr3h4onsdhjojrck7tqysrmnduzhgzdhf3l5uqake6w6474vvuha",
+      "width": 500,
+      "height": 500,
+      "format": "png"
```
Comment (macvts) [inline _data/icons/polynomialSepolia.json:6]: same here, change to jpg

**Review by macvts — COMMENTED** [review]
as mentioned

**Comment by macvts** [comment]
Screenshot 2567-08-04 at 03 09 44

try syncing your forked repo once again to update all the updated commits on original branch @Aniket965

**Comment by Aniket965** [comment]
done @macvts

**Comment by Aniket965** [comment]
yo @ligi bro, can you review this ?

**Review by ligi — APPROVED** [review]

**Pull request merged**